Microsoft.Toolkit.Uwp.UI.Controls/ColorPickerButton/WindowsColorPalette.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Color = Windows.UI.Color; // Type can be changed to CoreColor, etc.
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Implements the standard Windows 10 color palette.
+    /// </summary>
+    public class WindowsColorPalette : IColorPalette
```

Comment:
Was there a sample using this? I didn't see it. Would be nice to add.

Reply:
This is the default palette in all ColorPickerButtons so it's visible in all the samples if you click the palette tab. I'm not sure what you mean by this? You wan't a separate sample showing off various color palette's? I don't have that kind of time to spend on this :)

As mentioned it's a modification I made of the existing Windows 10 settings app colors for reasons documented in code.